source/core_sntp_client.c

```diff
@@ -32,9 +32,18 @@
 /* SNTP client library API include. */
 #include "core_sntp_client.h"
 
+/**
+ * @brief Utility to convert fractions part of SNTP timestamp to milliseconds.
+ *
+ * @param[in] fractions The fractions value in an SNTP timestamp.
+ */
+#define FRACTIONS_TO_MS( fractions ) \
+    ( fractions / ( SNTP_FRACTION_VALUE_PER_MICROSECOND * 1000U ) )
+
 SntpStatus_t Sntp_Init( SntpContext_t * pContext,
                         const SntpServerInfo_t * pTimeServers,
                         size_t numOfServers,
+                        uint32_t serverResponseTimeoutMs,
                         uint8_t * pNetworkBuffer,
                         size_t bufferSize,
                         SntpResolveDns_t resolveDnsFunc,
```

Comment:
As a caller of this function, it's hard for me to tell how each parameter is related to one another. Might it be worth grouping parameters into separate structs to make the relationships self-documenting.

Reply:
That's a good point, can take that hygiene in a separate PR 🙂